patches/MAGETWO-53941__fix_enterprise_braintree_code__100.1.6.patch

```diff
@@ -0,0 +1,50 @@
+<+>UTF-8
```

Comment:
@NadiyaS  shouldn't this part be removed from https://github.com/magento/magento-cloud-patches/blob/7beeb00e34d607471c527b00cfe9d0803fbac04c/patches/MAGETWO-53941__fix_enterprise_paypal_codes__100.1.6.patch as you separated it?

Also, why we started adding encoding into patches? Never seen UTF-8 in patches before.

Reply:
@YPyltiai thank you for pointing this out. I did not push that commit where I removed this and updates in the patch for PP